Gemfile

```diff
@@ -47,6 +47,10 @@ gem 'whenever', '~>0.6.2', :require => false
 gem 'nokogiri', '>= 1.7.1'
 gem 'mechanize'
 gem 'sanitize'
+# Until there is a working 
```

Comment:
The gem change is good, but I think you forgot to complete your sentence here. Do you mean something like, 
```
Until there is a working solution to
https://otwarchive.atlassian.net/browse/AO3-4957
```